spec/rubocop/server/rubocop_server_spec.rb

```diff
@@ -21,6 +21,10 @@
       end
 
       it 'displays a restart information message' do
+        # FIXME: Avoid flaky test for RSpec 4. It may be related that test-queue is not available.
+        # https://github.com/rubocop/rubocop/pull/10806#discussion_r918415067
+        skip if ENV['GITHUB_JOB'] == 'rspec4'
```

Comment:
FYI while the rspec4 job is the one that fails the most, I've seen this fail in other runs as well.

Reply:
Yes. It will probably take some time to investigate. It is targeted to the CI job that occur the most first.